libexec/tgenv-list-remote

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
-pages=4
+set -e
+
+link_release="https://api.github.com/repos/gruntwork-io/terragrunt/tags?per_page=1000"
+
+# the  response header (--head) contains a link to the last page, contains the last page number and the "last" keyword to help identify
+page_curl=$(curl -s --tlsv1.2 --head "$link_release")
+only_relevant_part=$(echo "$page_curl" | grep -o -E "&page=[0-9]+>; rel=\"last")
```

Comment:
`only_relevant_part` relevant to who?
please find better name

Reply:
9ab31e55